File: python-program/multithreading.py
from threading import * #to inherit thread class
from time import * #this is used to implement sleep

class Hello(Thread): #inherits thread class
    def run(self):  # use run method while creating threads
        for i in range(5):
            print("HELLO AYUSH")
            sleep(1)

class Hi(Thread):
    def run(self):
        for i in range(5):
            print("HI AYUSH")
            sleep(1)

o1 = Hello()
o2 = Hi()

o1.start()
sleep(0.2) #so that these two threads donot collide 
o2.start()
o1.join()   #after using join  next execution will only start only when the thread has completed execution 
o2.join()

print("Finish")

File: java program/multithreading/src/multithreading/Multithreading.java

package multithreading;
class threadeg extends Thread
{
    public void run()
    {
        for(int i =0;i<=5;i++)
        {
            try{
                Thread.sleep(500);
            }catch(Exception e)
            {
                System.out.println(e);
            }
            System.out.println(i);
        }
    }
}

public class Multithreading {

    
    public static void main(String[] args) {
        threadeg t1 = new threadeg();
        threadeg t2 = new threadeg();
        t1.start();
        t2.start();
        System.out.println(t2.getPriority());
        t2.setPriority(7);
        System.out.println(t2.getPriority());
        System.out.println(t1.getName());
        t1.setName("Ayush");
        System.out.println(t1.getName());
    }
    
}
